diff --git a/components/flags.ts b/components/flags.ts
index 0fb6eb6..28da165 100644
--- a/components/flags.ts
+++ b/components/flags.ts
@@ -34,7 +34,7 @@ export const defaultFlags: Flags = {
                 category: "Gameplay",
                 title: "gameplayUnlockAllShortcuts",
                 desc: "When set to true, all shortcuts will always be unlocked.",
-                default: false,
+                default: true,
             },
             gameplayUnlockAllFreelancerMasteries: {
                 category: "Gameplay",
@@ -53,7 +53,7 @@ export const defaultFlags: Flags = {
                 category: "Gameplay",
                 title: "enableMasteryProgression",
                 desc: "When set to false, mastery progression will be disabled and all unlockables will be awarded at the beginning",
-                default: true,
+                default: false,
             },
             enableIsolatedUnlockables: {
                 category: "Gameplay",
@@ -102,13 +102,13 @@ export const defaultFlags: Flags = {
                 category: "Services",
                 title: "leaderboards",
                 desc: "Allow your times to be submitted to the ingame leaderboards. If you do not want your times on the leaderboards, change this to false.",
-                default: true,
+                default: false,
             },
             updateChecking: {
                 category: "Services",
                 title: "updateChecking",
                 desc: "Allow Peacock to check for updates on startup.",
-                default: true,
+                default: false,
             },
             loadoutSaving: {
                 category: "Services",
