diff --git a/components/candle/progressionService.ts b/components/candle/progressionService.ts
index e5d5952d..90ba7976 100644
--- a/components/candle/progressionService.ts
+++ b/components/candle/progressionService.ts
@@ -40,6 +40,7 @@ import {
 import { writeUserData } from "../databaseHandler"
 import { MasteryPackageDrop } from "../types/mastery"
 import assert from "assert"
+import { getFlag } from "../flags"
 
 export class ProgressionService {
     // NOTE: Official will always grant XP to both Location Mastery and the Player Profile
@@ -52,6 +53,7 @@ export class ProgressionService {
         location: string,
         subPackage?: string,
     ): void {
+        if (getFlag("enableMasteryProgression")) {
         // Total XP for profile XP is the total sum of the action and mastery XP
         const xp = actionXp + masteryXp
 
@@ -84,6 +86,7 @@ export class ProgressionService {
 
         // Saves profile data
         writeUserData(userProfile.Id, contractSession.gameVersion)
+        }
     }
 
     // Returns current mastery progression for given location
