diff --git a/components/menus/menuSystem.ts b/components/menus/menuSystem.ts
index bb7db167..b4c51953 100644
--- a/components/menus/menuSystem.ts
+++ b/components/menus/menuSystem.ts
@@ -104,7 +104,6 @@ export class MenuSystemDatabase {
                 }
 
                 configs.push("menusystem/data/ishitman3available.json")
-                configs.push("menusystem/pages/hub/modals/roadmap/modal.json")
                 configs.push(
                     "menusystem/pages/hub/data/isfullmenuavailable.json",
                 )
