diff --git a/components/inventory.ts b/components/inventory.ts
index be4a49e..063fb2c 100644
--- a/components/inventory.ts
+++ b/components/inventory.ts
@@ -29,6 +29,7 @@ import {
     H1_GOTY_UNLOCKABLES,
     H1_REQUIEM_UNLOCKABLES,
     H2_RACCOON_STINGRAY_UNLOCKABLES,
+    H2_COLLECTORS_UNLOCKABLES,
     LAMBIC_UNLOCKABLES,
     MAKESHIFT_UNLOCKABLES,
     PENICILLIN_UNLOCKABLES,
@@ -74,6 +75,7 @@ const DELUXE_DATA = [
     ...H1_GOTY_UNLOCKABLES,
     ...H1_REQUIEM_UNLOCKABLES,
     ...H2_RACCOON_STINGRAY_UNLOCKABLES,
+    ...H2_COLLECTORS_UNLOCKABLES,
     ...MAKESHIFT_UNLOCKABLES,
     ...SIN_ENVY_UNLOCKABLES,
     ...SIN_GLUTTONY_UNLOCKABLES,
@@ -84,6 +86,7 @@ const DELUXE_DATA = [
     ...SIN_WRATH_UNLOCKABLES,
     ...TRINITY_UNLOCKABLES,
     ...WINTERSPORTS_UNLOCKABLES,
+    ...SMART_CASUAL_UNLOCKABLES,
     ...SAMBUCA_UNLOCKABLES,
     ...PENICILLIN_UNLOCKABLES,
     ...PENICILLIN_ITEMS_UNLOCKABLES,
@@ -428,6 +431,15 @@ export class InventoryService {
                     )
                 }
 
+                if (H2_COLLECTORS_UNLOCKABLES.includes(id)) {
+                    return (
+                        e.includes("6408de14f7dc46b9a33adcf6cbc4d159") ||
+                        e.includes("afa4b921503f43339c360d4b53910791") ||
+                        e.includes("84a1a6fda4fb48afbb78ee9b2addd475") || // WoA Deluxe
+                        e.includes("1829590")
+                    )
+                }
+
                 if (SMART_CASUAL_UNLOCKABLES.includes(id)) {
                     return (
                         e.includes("6408de14f7dc46b9a33adcf6cbc4d159") ||
diff --git a/components/ownership.ts b/components/ownership.ts
index 9d6acf5..8b7aeb0 100644
--- a/components/ownership.ts
+++ b/components/ownership.ts
@@ -176,6 +176,11 @@ export const SMART_CASUAL_UNLOCKABLES = [
     "PROP_MELEE_BLACK_PHONE_CORD",
 ]
 
+export const H2_COLLECTORS_UNLOCKABLES = [
+    "TOKEN_OUTFIT_MIDNIGHT_BLACK_COLLECTORS",
+    "PROP_DEVICE_REMOTE_RUBBERDUCK_CONCUSSION_COLLECTORS",
+]
+
 export const DELUXE_UNLOCKABLES = [
     // dubai
     "PROP_CONTAINER_SUITCASE_GOLDEN",
