diff --git a/components/menuData.ts b/components/menuData.ts
index bd33b144..1e5bccae 100644
--- a/components/menuData.ts
+++ b/components/menuData.ts
@@ -658,7 +658,7 @@ function generateSelectPage(
                 unlockable.Properties.UnlockLevel =
                     controller.masteryService
                         .getMasteryForUnlockable(unlockable, gameVersion)
-                        ?.Level.toString() ?? "1"
+                        ?.Level.toString() ?? ""
                 return unlockable
             })
             .sort(unlockLevelComparer)
 
