diff --git a/components/index.ts b/components/index.ts
index 4771abd..b3c5f98 100644
--- a/components/index.ts
+++ b/components/index.ts
@@ -66,6 +66,7 @@ import {
     STEAM_NAMESPACE_2016,
     STEAM_NAMESPACE_2018,
     STEAM_NAMESPACE_2021,
+    STEAM_NAMESPACE_2021_DEMO,
     STEAM_NAMESPACE_SCPC,
 } from "./platformEntitlements"
 import { legacyProfileRouter } from "./2016/legacyProfileRouter"
@@ -194,7 +195,7 @@ app.get(
                 "pc-prod_6"
         }
 
-        if (req.query.issuer === STEAM_NAMESPACE_2021) {
+        if (req.query.issuer === STEAM_NAMESPACE_2021 || req.query.issuer ===  STEAM_NAMESPACE_2021_DEMO) {
             config.Versions[0].SERVER_VER.GlobalAuthentication.RequestedAudience =
                 "steam-prod_8"
         }
@@ -350,6 +351,7 @@ app.use(
                     break
                 case "fghi4567xQOCheZIin0pazB47qGUvZw4":
                 case STEAM_NAMESPACE_2021:
+                case STEAM_NAMESPACE_2021_DEMO:
                     req.serverVersion = "8-14"
                     break
                 default:
diff --git a/components/oauthToken.ts b/components/oauthToken.ts
index b1dd1a2..cebc342 100644
--- a/components/oauthToken.ts
+++ b/components/oauthToken.ts
@@ -25,6 +25,7 @@ import { log, LogLevel } from "./loggingInterop"
 import {
     STEAM_NAMESPACE_2018,
     STEAM_NAMESPACE_2021,
+    STEAM_NAMESPACE_2021_DEMO,
 } from "./platformEntitlements"
 import {
     getExternalUserData,
@@ -142,7 +143,8 @@ export async function handleOauthToken(
 
     const isHitman3 =
         external_appid === "fghi4567xQOCheZIin0pazB47qGUvZw4" ||
-        external_appid === STEAM_NAMESPACE_2021
+        external_appid === STEAM_NAMESPACE_2021 ||
+        external_appid === STEAM_NAMESPACE_2021_DEMO
 
     const gameVersion: GameVersion = isFrankenstein
         ? "scpc"
diff --git a/components/platformEntitlements.ts b/components/platformEntitlements.ts
index 2f544ed..2f73b47 100644
--- a/components/platformEntitlements.ts
+++ b/components/platformEntitlements.ts
@@ -129,6 +129,7 @@ export const STEAM_NAMESPACE_SCPC = "783781"
 export const STEAM_NAMESPACE_2018 = "863550"
 export const EPIC_NAMESPACE_2021 = "ed55aa5edc5941de92fd7f64de415793"
 export const STEAM_NAMESPACE_2021 = "1659040"
+export const STEAM_NAMESPACE_2021_DEMO = "1847520"
 
 export const FRANKENSTEIN_SNIPER_ENTITLEMENTS = [STEAM_NAMESPACE_2016, "783781"]
 
